### PR TITLE
IOS: add properties/methods for redundancy info

### PR DESCRIPTION
**Pull request by jmcgill298**

### CODE REVIEW

File: pyntc/devices/ios_device.py [hunk 1]
```diff
@@ -341,6 +395,56 @@ def handler(signum, frame):
         else:
             print("Need to confirm reboot with confirm=True")
 
+    @property
+    def redundancy_mode(self):
+        """
+        The oprating redundancy mode of the device.
```
Comment (mundruid) [inline pyntc/devices/ios_device.py:401]: Change "oprating" to "operating"

File: test/unit/test_devices/test_ios_device.py [hunk 2]
```diff
@@ -348,12 +348,121 @@ def test_enable_from_config(self):
...
+@pytest.mark.parametrize(
+    "redundancy_state,expected",
+    (
+        ("active", True),
+        ("standby hot", False),
+        (None, True),
+    ),
+    ids=("active", "standby_hot", "unsupported"),
+)
+def test_is_active(mock_redundancy_state, redundancy_state, expected, ios_device):
```
Comment (mundruid) [inline test/unit/test_devices/test_ios_device.py:361]: Should there be a `test_is_not_active` for the `False` return or would this be too trivial?

**Review by mundruid — CHANGES_REQUESTED** [review]

File: test/unit/test_devices/test_ios_device.py (hunk 2 above)
Comment (jmcgill298) [reply test/unit/test_devices/test_ios_device.py:361]: @mundruid that is covered by the second test case in the parametrize decorator.

File: pyntc/devices/ios_device.py (hunk 1 above)
Comment (jmcgill298) [inline pyntc/devices/ios_device.py:401]:
```suggestion
        The operating redundancy mode of the device.
```

**Comment by jmcgill298** [comment]
Thanks for the review @mundruid

**Pull request merged**